### PR TITLE
Add grid lines for activation images, fixes #9130

### LINKED ISSUE #9130: [docs] Grid lines for activation function plots
Without them it's hard to understand the right asymptote of sigmoid, and the linear regime bounds (more of a pedagogical thing for new users)

https://pytorch.org/docs/stable/nn.html?highlight=multilabel_soft_margin_loss#torch.nn.Sigmoid

Even very light lightgray dotted grid lines would be nice

### PR DESCRIPTION
**Pull request by vishwakftw**
1. Add dashed light blue line for asymptotes.
2. RReLU was missing the activation image.
3. make clean in docs will remove the activation images too.

Sample image:
image

### CODE REVIEW

**Comment by vishwakftw** [comment]
cc: @vadimkantorov

**Comment by SsnL** [comment]
Looks good. Btw, why do we have the gray lines?

**Comment by vishwakftw** [comment]
The overlay was added in an older PR, so that it made the transition look better. I actually find it a bit confusing. Do you think we should remove them?

**Comment by vadimkantorov** [comment]
what do you thing about adding vertical lines as well? to see clearly where breakpoints happen

**Comment by vishwakftw** [comment]
This is how it looks with breakpoints.
image

I have removed the overlay to avoid confusion.

**Comment by apaszke** [comment]
@vadimkantorov I would say that would be too much. We could add some extra ticks on the X axis, but lines are usually asymptotes, and this is not what's happening there. It also adds visual clutter.

**Comment by SsnL** [comment]
we can also just add the grid lines

**Comment by vishwakftw** [comment]
I agree with @apaszke , the vertical lines cause issues for plots like `ReLU6`.

**Comment by vishwakftw** [comment]
This is the current plot for HardTanh

image

Hope this is fine.

**Comment by karandwivedi42** [comment]
I think at least xticks and yticks on +1 and -1 should be there.

**Comment by vadimkantorov** [comment]
@apaszke @vishwakftw The last one looks good: conveys the meaning of asymptotes and enables grasping where the breakpoints are, if needed + not much visual clutter (but even the grid lines without asymptotes are fine for my taste).

**Comment by SsnL** [comment]
Thanks! Grid lines look a bit too subtle to me. What do you think?

Also, with grid lines, do we still want asymptotic lines?

**Comment by vishwakftw** [comment]
@SsnL removed asymptote lines and increased the width.

**Review by ezyang — APPROVED** [review]

**Review by ssnl — APPROVED** [review]
Thanks for doing this!

**Review by soumith — APPROVED** [review]
